### PR TITLE
Fixed APIMANAGER-3462 + improvements.

### PR DESCRIPTION
**Pull request by kasun32**
- fixed the scope specific access token generation issue where user had to select scopes each time they regenerate keys. (APIMANAGER-3462)
- fixed generated scopes not showing up until the page is refreshed.
- added a label to scopes for clarity in viewing.
- removed the copy button for scope names.

### CODE REVIEW

**Comment by kasun32** [comment]
commit message of 23dac57a57efb21d3d4fd258b107cd948e17dd0b is mistakenly entered as "password change feature". Please ignore. Only mentioned changes are done in this PR.

**Comment by harsha89** [comment]
@kasun32 so you can change the commit message with git commit --amend -m "new message" if tht's ur last commit.

File: components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java [hunk 1]
```diff
@@ -2745,6 +2745,7 @@ public static NativeObject jsFunction_getAllSubscriptions(Context cx,
                             appObj.put("prodKey", appObj, prodKey.getAccessToken());
 
 			                appObj.put("prodKeyScope", appObj, prodKeyScope);
+	                        appObj.put("prodKeyScopeValue", appObj, prodKey.getTokenScope());
```
Comment (harsha89) [inline components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java]: Small formatting issue.

**Comment by kasun32** [comment]
@harsha89 I commited via IDE so I didn't notice the mistake until the last commit. Couldn't change as more commits were done after it.

**Comment by harsha89** [comment]
@kasun32 you can renamed multiple commits. Refer https://help.github.com/articles/changing-a-commit-message/

**Pull request merged**